pinot-tools/src/main/resources/examples/batch/colocated/userGroups/userGroups_offline_table_config.json

```diff
@@ -1,6 +1,7 @@
 {
   "tableName": "userGroups",
   "tableType": "OFFLINE",
+  "isDimTable": true,
```

Comment:
Given we also use colocated quickstart to test colocated joins, I think we should have another table like userGroups that is not a dim table.

This is not an actual problem right now that both colocated and dim joins have to be enabled explicitly with hints, but will be an issue in the future when both are going to be applied by default.